src/test/java/uk/gov/pay/api/it/ResourcesFiltersRedisRateLimiterITest.java

```diff
@@ -0,0 +1,330 @@
+package uk.gov.pay.api.it;
+
+import com.google.common.collect.ImmutableMap;
+import com.jayway.restassured.response.ExtractableResponse;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.response.ValidatableResponse;
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.pay.api.it.fixtures.PaymentNavigationLinksFixture;
+import uk.gov.pay.api.model.Address;
+import uk.gov.pay.api.model.CardDetails;
+import uk.gov.pay.api.model.PaymentState;
+import uk.gov.pay.api.model.RefundSummary;
+import uk.gov.pay.api.utils.ChargeEventBuilder;
+import uk.gov.pay.api.utils.DateTimeUtils;
+import uk.gov.pay.api.utils.JsonStringBuilder;
+
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static javax.ws.rs.core.HttpHeaders.AUTHORIZATION;
+import static org.hamcrest.core.IsCollectionContaining.hasItem;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import static uk.gov.pay.api.it.fixtures.PaginatedPaymentSearchResultFixture.aPaginatedPaymentSearchResult;
+import static uk.gov.pay.api.it.fixtures.PaymentSearchResultBuilder.aSuccessfulSearchPayment;
+
+public class ResourcesFiltersRedisRateLimiterITest extends PaymentResourceITestBase {
```

Comment:
given RateLimiterFilter determines if a request is a GET, i think testing all the get methods, and testing the rate limit and invalid authorization header for each method is probably overkill. i think some tests could be removed from here.

Reply:
I think renaming files caused confusion here. I have restored ResourcesFiltersITest which is testing all public API endpoints for different errors returned. I now created only one class ResourcesFiltersLocalRateLimiterITest which do not access redis and few tests around it. All other tests use Redis by default.